flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/batch/connectors/hive/HiveTableSource.java

```diff
@@ -67,13 +67,15 @@
 	private List<Map<String, String>> partitionList = new ArrayList<>();
 	private Map<Map<String, String>, HiveTablePartition> partitionSpec2HiveTablePartition = new HashMap<>();
 	private boolean initAllPartitions;
+	private boolean applyPartitionPrunning;
```

Comment:
I'm a little confused about the name of this field. Is it about "whether we should apply partition pruning", or "whether partition pruning has been applied"? If it's the latter, I think we can rename it to something like `partitionPruned`.